src/plugins/inputs/otr.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function(network, chan, cmd, args) {
+	if (cmd !== "otr") {
+		return;
+	}
+	if (args.length === 0 || args[0] === "") {
```

Comment:
I'm not sure. [Here](https://github.com/astorije/shout/blob/2cee0ea6ef5ee51de0190332f976934b55bbc8e4/src/plugins/inputs/msg.js#L7) and [there](https://github.com/astorije/shout/blob/2cee0ea6ef5ee51de0190332f976934b55bbc8e4/src/plugins/inputs/services.js#L12) are the only 2 places where I could find it in `plugins/inputs`. I am tempted to leave it, for completion... I don't know under what conditions we activate the first check or the second one.